util/multisig/src/error.rs

```diff
@@ -0,0 +1,36 @@
+use failure::Context;
+
+#[derive(Debug)]
+pub struct Error {
+    inner: Context<ErrorKind>,
+}
+
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Fail)]
+pub enum ErrorKind {
+    #[fail(display = "The count of sigs should less than pks.")]
+    SigCountOverflow,
+    #[fail(display = "The count of sigs less than threshold.")]
+    SigNotEnough,
+    #[fail(display = "Threshold not reach, sigs: {} required sigs: {}.", _0, _1)]
+    Threshold(usize, usize),
```

Comment:
I personally dislike the tuple like structure when there are multiple fields, and the meaning of each field is not clear.